Parsers/ZoomReports/Zoom.txt

```diff
@@ -0,0 +1,107 @@
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias (e.g. Zoom).
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. Zoom | take 10).
+// References : 
+// Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+// Tech Community Blog on KQL Functions : https://techcommunity.microsoft.com/t5/Azure-Sentinel/Using-KQL-functions-to-speed-up-analysis-in-Azure-Sentinel/ba-p/712381
+//
+let Zoom_dates_report_view = view () {
+    Zoom_CL 
+    | where event_type_s == "dates"
```

Comment:
Usually good to use "=~" to allow for the string value match to not be case sensitive

Reply:
Thanks, but in this case, it is not necessary because data types exactly predefined in the python code.